tox.ini

```diff
@@ -100,12 +100,14 @@ passenv=
     USERNAME
     CI_PYTEST_SPLIT_ARGS
 
-[testenv:{,core-}{,notebooks-}py{37,38,39,310,311,312}]
+[testenv:{,core-}{,notebooks-}py{38,39,310,311,312}]
 install_command=
     ; pytorch installations on non-darwin need the `-f`
     pip install --timeout 600 --extra-index-url https://download.pytorch.org/whl/cpu {opts} {packages}
 deps=
     {[base]deps}
+    .[reports]
+    polyfactory
```

Comment:
one other option, if we want to keep the py37 for the sdk tests.
we can have a label here as it only used for the reports tests anyways... 
(not saying we need to do it, but an option)

Reply:
i could add another factor, but i'd also have to adjust the test paths selected by the CI; i think the additional complexity might not be worth it